### ISSUE TITLE
choosealicence.com has an invalid TLS certificate

### ISSUE THREAD

**Issue by aapeliv** [github/choosealicense.com#712]
choosealicence.com (note the second "c" in "licence") that redirects to choosealicense.com has the certificate for github.com, which brings up a warning screen

**Comment by mlinksva** [comment]
True, choosealicen**c**e.com does have the certificate for github.com, though I don't get a warning screen on either current Firefox 71.0 or Chrome 79.0.3945.86. What software is bringing up a warning screen for you @aapeliv?

cc @lgarron who might plausibly have some insight on how much of a problem this is and whether it is feasible to correct at this time.

**Comment by mlinksva** [comment]
Ah, http://choosealicence.com redirects to https://choosealicense.com therefore no warning screen, but https://choosealicence.com obtains warning screen.

**Comment by aapeliv** [comment]
I get the warning on Firefox (both latest stable as all as dev edition), and mobile Safari (iOS 13.3).

**Comment by TechnicianLP** [comment]
Adding a "www." to the url (ie. https://www.choosealicense.com/ ) causes the warning screen as well

**Comment by Tarekchehahde** [comment]
Opened https://github.com/github/choosealicense.com/pull/1370 — choosealicence.com typo notice on about page.
